src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+public class Task {
+    private String desc;
+    private int status; // 0 incomplete, 1 complete
+
+    public Task(String d, int s) {
+        this.desc = d;
+        this.status = s;
+    }
+
+    public void update(int i) {
+        this.status = i;
+    }
+
+    public void flip() {
+        if (this.status == 0) {
+            this.status = 1;
+        } else {
+            this.status = 0;
+        }
+    }
+
+    public int getStatus() {
+        return this.status;
+    }
+
+    public String getDesc() {
+        return this.desc;
+    }
+
+    public String statusMessage() {
```

Comment:
The naming for methods needs to be verb. For instance, getName().
```suggestion
    public String getStatusMessage() {
```
```suggestion
    public String returnStatusMessage() {
```

Reply:
Thanks for the feedback, have taken it into account when I finished some major refactoring today. Will continue on ironing out the issues.